src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMBuilderTest.java

```diff
@@ -711,6 +731,7 @@ public void given__cloud_branch_norev_userkey__when__build__then__scmBuilt() thr
 
     @Test
     public void given__server_branch_rev_anon__when__build__then__scmBuilt() throws Exception {
+        org.junit.Assume.assumeTrue(StringUtils.isBlank(source.rawUrl));
```

Comment:
Why do these tests not work if configured with a raw URL? The behavior of the class should be the same no matter how it is configured, right?